keeper-gradle-plugin/src/main/java/com/slack/keeper/VariantClasspathJar.kt

```diff
@@ -44,9 +48,29 @@ public abstract class BaseKeeperJarTask : DefaultTask() {
   @get:Input
   public abstract val emitDebugInfo: Property<Boolean>
 
+  private lateinit var artifacts: ArtifactCollection
+
   @get:OutputDirectory
   public abstract val diagnosticsOutputDir: DirectoryProperty
 
+  /**
+   * This artifact collection is the result of resolving the compilation classpath.
+   */
+  public fun setArtifacts(artifacts: ArtifactCollection) {
+    this.artifacts = artifacts
+  }
+
+  /**
+   * This is the "official" input for wiring task dependencies correctly, but is otherwise
+   * unused. This needs to use [InputFiles] and [PathSensitivity.ABSOLUTE] because the path to the
```

Comment:
kdoc is a little misleading. You actually _are_ using this input in your task action. In the code snippet I sent you, I wasn't using this input, but rather using the `artifacts` property, which is why I wrote the doc this way.